### DISCUSSION TITLE
Awk task

### DISCUSSION THREAD

**Discussion by Vlek** [wolandark/bash-dungeon#11] [Ideas]
Another helpful cli tool is awk, especially for structured data.

Maybe have it so that the user needs to count the sum of lines if it has a certain value in another column?

So have another vendor with a bunch of weapons. They have to count how many are slashing-based attack types?

Something like:
```
name, cost, type
halberd, 10, slashing
mace, 25, bashing
short sword, 3, slashing
long sword, 7, slashing
spear, 11, piercing
bow, 8, piercing
```

And then if column 3 == slashing, tally += column 2.

**Comment by wolandark** [comment]
I have a similar idea for adding awk. A vendor who has a messy index of items, they have to be sorted and properly formated and written to a new file.
